### PR TITLE
adding linting for vuejs

### PR DESCRIPTION
**Pull request by justvr**
As official TS attitude is to go to ESLint direction, I moved the linting to ESLint. https://eslint.org/blog/2019/01/future-typescript-eslint

---------overcame with 'htmlWhitespaceSensitivity': 'ignore' rule---------
VScode: unfortunately not everything can be fixed within the editor itself. 
For example: default.vue l58
```
<v-snackbar v-model="countHasBeenUpdated" :timeout="3000" bottom right
  >count has been changed</v-snackbar
>
```
This unpleasant formatting comes from new Prettier setup for Vuejs, and let's hope it will go away in the future or we'll just have to get used to it.
---------/overcame with 'htmlWhitespaceSensitivity': 'ignore' rule---------

**TODO**
Currently there is one problem addressing type `any`, which is opposite to TS purpose :) https://github.com/bradzacher/eslint-plugin-typescript/blob/master/docs/rules/no-explicit-any.md
honestly, wanted to find a way to turn it off, but then realised the rule is supported in compiler as well under the flag `--noImplicitAny` which confirms the importance of fixing it...
so could someone help me fixing it please?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "now-build": "nuxt build --spa"
   },
   "dependencies": {
+    "eslint-plugin-vue": "5.2.2",
```
Comment (lautr) [inline package.json]: isn't this a dev dependency?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    'plugin:@typescript-eslint/recommended',
+    'prettier/@typescript-eslint',
+    'plugin:prettier/recommended',
+    'plugin:vue/recommended',
+    '@vue/prettier',
+    '@vue/typescript'
+  ],
+  rules: {
+  }
+}
```
Comment (lautr) [inline .eslintrc.js]: missing new line

File: package.json (hunk 1 above)
Comment (justvr) [reply package.json]: yes, i'll change it

**Comment by lautr** [comment]
changes have been manually aligned with the nuxt 2.9.1 update

**Pull request closed without merging**